docs/migrate.ipynb

```diff
@@ -254,7 +254,45 @@
       "source": [
         "## Converting your code\n",
         "\n",
-        "Converting existing `tf.summary` usage to the TF 2.x API cannot be reliably automated, so the [`tf_upgrade_v2` script](https://www.tensorflow.org/guide/upgrade) just rewrites it all to `tf.compat.v1.summary`.  To migrate to TF 2.x, you'll need to adapt your code as follows:"
+        "Converting existing `tf.summary` usage to the TF 2.x API cannot be reliably automated, so the [`tf_upgrade_v2` script](https://www.tensorflow.org/guide/upgrade) just rewrites it all to `tf.compat.v1.summary` and will not enable the TF 2.x behaviors automatically."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1972f8ff0073"
+      },
+      "source": [
+        "### Partial Migration\n",
+        "\n",
+        "It is possible to invoke TF 2.x behaviors within <a href=\"https://www.tensorflow.org/api_docs/python/tf/compat/v1/summary\"><code>tf.compat.v1.summary</code></a> APIs under the following conditions:\n",
```

Comment:
I might add one or two extra sentences here to explain why the partial migration would be useful, and also to still encourage the full migration.  E.g. something like:

> To make migration to TF 2.x easier for users of model code that still depends heavily on the TF 1.x summary API logging ops like `tf.compat.v1.summary.scalar()`, it is possible to migrate only the writer APIs first, allowing for individual TF 1.x summary ops inside your model code to be fully migrated at a later point.
>
> To support this style of migration, tf.compat.v1.summary logging APIs will automatically forward to their TF 2.x equivalents under the following conditions:

Reply:
Done. Thanks!